packages/blade/src/components/Input/BaseInput/BaseInputWrapper copy.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
```

Comment:
Is this a redundant file? There's `BaseInputWrapper copy.tsx` as well as `BaseInputWrapper.tsx`

Reply:
yes by mistake, have removed in the next TextInput PR